app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      def create
+        book_suggestion = BookSuggestion.create!(book_suggestions_params)
+        render json: book_suggestion, adapter: :json, status: :created
```

Comment:
```suggestion
        render json: BookSuggestion.create!(book_suggestions_params), status: :created
```

Reply:
you should get the user_id from the current_user if there's one, but i don't care at this point :P